File: VB/MVVMDemo.Wpf/Modules/TypedStyles/DXBindingAndDXCommandViewModel.vb
Imports DevExpress.Mvvm
Imports System.Windows

Namespace MVVMDemo.TypedStylesDemo
    Public Class DXBindingAndDXCommandViewModel
        Inherits BindableBase

        Public Property FirstName() As String
            Get
                Return GetProperty(Function() FirstName)
            End Get
            Set(ByVal value As String)
                SetProperty(Function() FirstName, value)
            End Set
        End Property
        Public Property LastName() As String
            Get
                Return GetProperty(Function() LastName)
            End Get
            Set(ByVal value As String)
                SetProperty(Function() LastName, value)
            End Set
        End Property
        Public Sub Register()
            MessageBox.Show("Registered")
        End Sub
        Public Sub New()
            FirstName = "Gregory"
            LastName = "Price"
        End Sub
    End Class
End Namespace

File: CS/MVVMDemo.Wpf/Modules/TypedStyles/DXBindingAndDXCommandViewModel.cs
using DevExpress.Mvvm;
using System.Windows;

namespace MVVMDemo.TypedStylesDemo {
    public class DXBindingAndDXCommandViewModel : BindableBase {
        public string FirstName {
            get { return GetProperty(() => FirstName); }
            set { SetProperty(() => FirstName, value); }
        }
        public string LastName {
            get { return GetProperty(() => LastName); }
            set { SetProperty(() => LastName, value); }
        }
        public void Register() {
            MessageBox.Show("Registered");
        }
        public DXBindingAndDXCommandViewModel() {
            FirstName = "Gregory";
            LastName = "Price";
        }
    }
}
